IS7024/Pages/LudlowGarage.cshtml

```diff
@@ -6,72 +6,63 @@
     var sun = ViewData["sun"];
 }
 
-    <div class="text-center">
-        <h1 class="display-4">Upcoming Concerts at the Ludlow Garage</h1>
-        <p>Cincinnati, Ohio</p>
-        <hr>
-        <table border="1" width="90%">
-            <tr>
-                <th>Date</th>
-                <th>Artist</th>
-                <th>Status</th>
-                <th>Start Time</th>
-                <th>Sunset Time</th>
-                <th>Buy tickets</th>
-            </tr>
+<div class="text-center">
+    <h1 class="display-4">Upcoming Concerts at the Ludlow Garage</h1>
+    <p>Cincinnati, Ohio</p>
+    <hr>
+    <table border="1" width="90%">
+        <tr>
+            <th>Date</th>
+            <th>Artist</th>
+            <th>Status</th>
+            <th>Start Time</th>
+            <th>Sunset Time</th>
+            <th>Buy tickets</th>
+        </tr>
             @{
 
-    foreach (QuickType.Event eventss in events)
-    {
+                foreach (QuickType.Event eventss in events)
+                {
 
-        string date = eventss.Start.Date.ToString();
-        string[] dars = date.Split('/');
-        string day = dars[0];
-        string month = dars[1];
-        string year = dars[2];
-        string[] year1 = year.Split(' ');
-        string yearclean = year1[0];
+                    string date = eventss.Start.Date.ToString();
+                    string[] dateSplit = date.Split('/');
+                    string day = dateSplit[0];
+                    string month = dateSplit[1];
+                    string year = dateSplit[2];
+                    string[] year1 = year.Split(' ');
+                    string yearclean = year1[0];
 
 
-        if (eventss.Start.Datetime != null)
+                    if (eventss.Start.Datetime != null)
 
```

Comment:
The default spacing on my VS code must be different from yours because it's marking these changes in spacing that I did not make.